conf/resources/countrySynonyms.json

```diff
@@ -0,0 +1,7 @@
+{
+  "GB": {
+    "synonyms": [
+        "England", "Scotland", "Wales"
```

Comment:
Should there be a Northern Ireland synonym, or has this intentionally been omitted?

Reply:
GB doesn't technically include Northern Ireland, but in countriesEN its official name is listed as "The United Kingdom of Great Britain and Northern Ireland". Looking [here](https://find-and-update.company-information.service.gov.uk/advanced-search/get-results?companyNameIncludes=&companyNameExcludes=&registeredOfficeAddress=Northern+Ireland&incorporationFromDay=&incorporationFromMonth=&incorporationFromYear=&incorporationToDay=&incorporationToMonth=&incorporationToYear=&sicCodes=&dissolvedFromDay=&dissolvedFromMonth=&dissolvedFromYear=&dissolvedToDay=&dissolvedToMonth=&dissolvedToYear=) there is a menagerie of different combinations.

Reply:
I'd be up for adding "Northern Ireland" to the GB synonyms though as it's bound to happen.